src/app/dashboard/top-chart/top-chart.component.ts

```diff
@@ -7,66 +9,29 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./top-chart.component.scss']
 })
 export class TopChartComponent implements OnInit {
+  public totalXp: number = 0;
+  public maxWidth: number = 160;
 
-  public totalXp: number = 0
-  public maxWidth: number = 160
+  users: User[] = [];
```

Comment:
public access modifier is default. and you do not need to provide it explicitly. if you have any private function/propertiies (ones that are not used in the template) - for them you need to provide 'private' access modifier